backend/execution_cache/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Dockerfile for building the source code of execution_cache
```

Comment:
can you follow the other backend components, and put source code under backend/src and name Dockerfiles following Dockerfile.component convension. 

https://github.com/kubeflow/pipelines/tree/master/backend